src/app/(styles)/404.module.css

```diff
@@ -33,12 +35,13 @@
   color: var(--turquoise);
 }
 
-@media (width >= 720px) {
+@media (width >=45rem) {
   .icon-404 {
-    width: 380px;
-    height: 380px;
+    width: 23.75rem;
+    height: 23.75rem;
   }
+
   /* .continer-text-notFound {
-    margin-top: -60px;
+    margin-top: -3.75rem;
   } */
-}
+}
```

Comment:
Do we need these commented lines?

Reply:
It was a vision I had not reviewed. I didn't know it then, they already assigned me a card to work on it.

Reply:
vision ❌
view ✅
@SthefaniaEcheverry